Update roadmap for 2019 Q4 (#1196)
kubeedge/kubeedge: docs/getting-started/roadmap.md

The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.

KubeEdge 1.1 is our current stable branch. The roadmap below outlines new features that will be added to KubeEdge.

## 2019 Q4 Roadmap
- Support HA for cloudcore
- Support exec&logs API for edge application
- Support reliable message delivery from cloud to edge.

> - Replace data exchange format between cloud and edge from json to protobuf.

Reviewer: ```suggestion
- Add protobuf support for data exchange format between cloud and edge
```
